generators/docker-compose/generator.ts

```diff
@@ -374,15 +355,15 @@ export default class DockerComposeGenerator extends BaseWorkspacesGenerator<Base
     let warningMessage = 'To generate the missing Docker image(s), please run:\n';
     applications.forEach(application => {
       if (application.buildToolGradle) {
-        imagePath = this.destinationPath(workspaces.directoryPath, application.appFolder, 'build/jib-cache');
+        imagePath = this.destinationPath(this.directoryPath, application.appFolder, 'build/jib-cache');
```

Comment:
Would prefer to have `directoryPath` not part of `this`

Reply:
later